stylelint.config.js

```diff
@@ -1,3 +1,10 @@
 module.exports = {
-    extends: '@edx/stylelint-config-edx'
+    extends: '@edx/stylelint-config-edx',
+    rules: {
+        'selector-anb-no-unmatchable': null, // Disable the unknown rule
+        'no-descending-specificity': null,
+        'declaration-block-no-duplicate-properties': [true, {
+            ignore: ['consecutive-duplicates']
+        }]
+    }
```

Comment:
Since we have the violation threshold implemented again, we do not need to ignore these rules, correct?